core/utils/device/adb.py

```diff
@@ -170,3 +170,14 @@ def open_home(device_id):
         cmd = 'shell am start -a android.intent.action.MAIN -c android.intent.category.HOME'
         Adb.__run_adb_command(command=cmd, device_id=device_id)
         Log.info('Open home screen of {0}.'.format(str(device_id)))
+
+    @staticmethod
+    def install(apk_path, emulator_info):
```

Comment:
In Adb class we usually use id directly instead of emulator info (we may also use the same for real devices and emulator_info does not sound correct in this case).

Reply:
:+1: